pkg/authorization/toplevel_org_authorizer.go

```diff
@@ -86,8 +86,8 @@ func (a *topLevelOrgAccessAuthorizer) Authorize(ctx context.Context, attr author
 		return a.delegate.Authorize(ctx, attr)
 	}
 
-	cluster, err := genericapirequest.ValidClusterFrom(ctx)
-	if err != nil || cluster == nil || cluster.Name.Empty() {
+	cluster := genericapirequest.ClusterFrom(ctx)
+	if cluster == nil || cluster.Name.Empty() {
 		kaudit.AddAuditAnnotations(
 			ctx,
 			TopLevelContentAuditDecision, DecisionNoOpinion,
```

Comment:
(next line) since the error is gone, this should now be:
```
TopLevelContentAuditReason, "empty cluster name",
```

Reply:
updated, thx